app/models/manageiq/providers/azure/cloud_manager/vm.rb

```diff
@@ -2,6 +2,32 @@ class ManageIQ::Providers::Azure::CloudManager::Vm < ManageIQ::Providers::CloudM
   include_concern 'Operations'
   include_concern 'ManageIQ::Providers::Azure::CloudManager::VmOrTemplateShared'
 
+  # Only select the columns we actually use for the Azure provider.
+  default_scope do
+    select([
+      :availability_zone_id,
+      :cloud,
+      :created_on,
+      :description,
+      :ems_id,
+      :ems_ref,
+      :flavor_id,
+      :guid,
+      :location,
+      :name,
+      :orchestration_stack_id,
+      :power_state,
+      :publicly_available,
+      :raw_power_state,
+      :resource_group_id,
+      :state_changed_on,
+      :tenant_id,
+      :type,
+      :uid_ems,
+      :vendor
+    ])
+  end
```

Comment:
This seems like a really bad idea.  If we add a core column we have to remember to add it here.  What kind of *actual* performance gain / memory improvments are we talking about?

cc @bdunne @agrare